packages/gatsby-source-wordpress/src/gatsby-node.js

```diff
@@ -88,111 +91,190 @@ exports.sourceNodes = async (
+    {
+      name: `generateFakeWordpressId`,
+      normalizer: ({ entities }) => normalize.generateFakeWordpressId(entities),
+    },
+    // Remove ACF key if it's not an object, combine ACF Options
+    {
+      name: `normalizeACF`,
+      normalizer: ({ entities }) => normalize.normalizeACF(entities),
+    },
+    // Combine ACF Option Data entities into one but split by IDs + options
+    {
+      name: `combineACF`,
+      normalizer: ({ entities }) => normalize.combineACF(entities),
+    },
+    // Creates entities from object collections of entities
+    {
+      name: `normalizeEntities`,
+      normalizer: ({ entities }) => normalize.normalizeEntities(entities),
+    },
+    // Standardizes ids & cleans keys
+    {
+      name: `standardizeKeys`,
+      normalizer: ({ entities }) => normalize.standardizeKeys(entities),
+    },
+    // Converts to use only GMT dates
+    {
+      name: `standardizeDates`,
+      normalizer: ({ entities }) => normalize.standardizeDates(entities),
+    },
+    // Lifts all "rendered" fields to top-level.
+    {
+      name: `liftRenderedField`,
+      normalizer: ({ entities }) => normalize.liftRenderedField(entities),
+    },
+    // Exclude entities of unknown shape
+    {
+      name: `excludeUnknownEntities`,
+      normalizer: ({ entities }) => normalize.excludeUnknownEntities(entities),
+    },
+    // Creates Gatsby IDs for each entity
+    {
+      name: `createGatsbyIds`,
+      normalizer: ({ creteNodeId, entities, _siteURL }) =>
+        normalize.createGatsbyIds(createNodeId, entities, _siteURL),
+    },
+    // Creates links between authors and user entities
+    {
+      name: `mapAuthorsToUsers`,
+      normalizer: ({ entities }) => normalize.mapAuthorsToUsers(entities),
+    },
+    // Creates links between posts and tags/categories.
+    {
+      name: `mapPostsToTagsCategories`,
+      normalizer: ({ entities }) =>
+        normalize.mapPostsToTagsCategories(entities),
+    },
+    // Creates links between tags/categories and taxonomies.
+    {
+      name: `mapTagsCategoriesToTaxonomies`,
+      normalizer: ({ entities }) =>
+        normalize.mapTagsCategoriesToTaxonomies(entities),
+    },
+    // Normalize menu items
+    {
+      name: `normalizeMenuItems`,
+      normalizer: ({ entities }) => normalize.normalizeMenuItems(entities),
+    },
+    // Creates links from entities to media nodes
+    {
+      name: `mapEntitiesToMedia`,
+      normalizer: ({ entities }) => normalize.mapEntitiesToMedia(entities),
+    },
+    // Downloads media files and removes "sizes" data as useless in Gatsby context.
+    {
+      name: `downloadMediaFiles`,
+      normalizer: async ({
+        entities,
+        store,
+        cache,
+        createNode,
+        createNodeId,
+        touchNode,
+        getNode,
+        auth,
+        reporter,
+        keepMediaSizes,
+      }) =>
+        await normalize.downloadMediaFiles({
```

Comment:
maybe can be shorter?

```suggestion
        async(helpers) => await normalize.downloadMediaFiles(helpers)
```